crates/matrix-sdk/src/room/timeline/inner.rs

```diff
@@ -577,17 +612,28 @@ impl ProfileProvider for room::Common {
             }
         }
     }
+
+    async fn read_receipts_for_event(&self, event_id: &EventId) -> IndexMap<OwnedUserId, Receipt> {
+        match self.event_receipts(ReceiptType::Read, ReceiptThread::Unthreaded, event_id).await {
+            Ok(receipts) => receipts.into_iter().collect(),
+            Err(e) => {
+                error!(%event_id, "Failed to get read receipts for event: {e}");
```

Comment:
We're now using `Debug` formatting for tracing string fields everywhere.

```suggestion
                error!(?event_id, "Failed to get read receipts for event: {e}");
```